### ISSUE TITLE
In admin webInterface,  No "awsAccessKey", "awsSecretKey"

### ISSUE THREAD

**Issue by poulsp** [project-alice-assistant/ProjectAlice#457]
As the title say.

**Comment by philipp2310** [comment]
First look, configTemplate.json has wrong parent conditions (tts = false, should be tts = amazon)
```
"awsAccessKey"            : {
		"defaultValue": "",
		"dataType"    : "string",
		"isSensitive" : true,
		"description" : "Your Amazon services access key",
		"category"    : "credentials",
		"parent"      : {
			"config"   : ["tts", "ttsFallback"],
			"condition": "is",
			"value"    : false
		}
	},
"awsSecretKey"            : {
		"defaultValue": "",
		"dataType"    : "string",
		"isSensitive" : true,
		"description" : "Your Amazon services secret key",
		"category"    : "credentials",
		"parent"      : {
			"config"   : ["tts", "ttsFallback"],
			"condition": "is",
			"value"    : false
		}
	}
```
will double check later and fix.
Thanks for the report!

**Comment by philipp2310** [comment]
fixed in 455bf26 for rc1
https://github.com/project-alice-assistant/ProjectAlice/commit/455bf26da28ffee9988bc9b71343a2b8d566da60
